src/shipping/pickup-option-reducer.spec.ts

```diff
@@ -25,14 +25,20 @@ describe('pickupOptionReducer()', () => {
         });
     });
 
-    it('returns a new state when pickup options are fetched', () => {
+    it.only('returns a new state when pickup options are fetched', () => {
+        const query = getQueryForPickupOptions();
         const action: LoadPickupOptionsAction = {
             type: PickupOptionActionType.LoadPickupOptionsSucceeded,
+            meta: query,
             payload: [getPickupOptions()],
         };
 
+        const codedKey = btoa(`${query.consignmentId}-${JSON.stringify(query.searchArea)}`);
```

Comment:
`JSON.stringify` doesn't sort the object keys. Therefore if the keys are in a different order, it will produce a different hash key. So you either want to sort them first, or just manually flatten the values yourself.

Reply:
May flatten it manually, since sorting can still be unreliable in this case.